docs/variants/pc_engines/test-matrix.md

```diff
@@ -0,0 +1,9 @@
+# Test matrix
+
+## About
+
+The test matrix is used to determine the scope of tests which the DUT is
+subjected from before the release of the new binary. The test scope is the same
+for all supported platforms of the PC Engines apu2/3/4/6.
+
+TBD
```

Comment:
We should just link to: https://docs.google.com/spreadsheets/d/1wSE6xA3K3nXewwLn5lV39_2wZL1kg5AkGb4mvmG3bwE/edit#gid=1670191276v for Dasharo (coreboot+UEFI)